Gemfile

```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'will_paginate', '~> 3.0.6'
```

Comment:
Hmm. Hadn't heard of this gem. I usually use kaminari. It might be worth asking in #backend if we have a preferred pagination gem.